### DISCUSSION TITLE
Does anyone know why I encounter errors when I set the PYNQ board to RFSoC4x2 in the practice examples?

### DISCUSSION THREAD

**Discussion by harrisonasd** [Xilinx/finn#992] [Q&A] [answered]
Does anyone know why I encounter errors when I set the PYNQ board to RFSoC4x2 in the practice examples?
![image](https://github.com/Xilinx/finn/assets/105301827/15e0d754-c488-415b-9bb2-0d05943e6603)
![image](https://github.com/Xilinx/finn/assets/105301827/2cb7f160-93e9-4f8d-89f1-26ddd00c9fbe)

**Comment by fpjentzsch** [comment]
Are you sure this is related to the board setting? It looks to me like your graph in the later build steps contains nodes which do not have any "backend" attribute when they should all belong to the "fpgadataflow" backend. Did you change anything else that could result in incorrect streamlining or HLS-mapping?

**Comment by harrisonasd** [comment]
indeed it was the issue you mentioned, but when I switched the board to rfsoc4x2, I encountered another problem
![image](https://github.com/Xilinx/finn/assets/105301827/6c1d44fd-45ca-4108-a149-3755dbc7dd37)
![image](https://github.com/Xilinx/finn/assets/105301827/f6f40121-430d-45ef-afc3-4d261b4923c9)

**Comment by auphelia** [comment, marked as answer]
@harrisonasd, I think your question was already answered here: https://github.com/Xilinx/finn/discussions/1011
Please reopen if necessary.
